### DISCUSSION TITLE
Blazor WASM Module Not Fully Functional in Non-Default Tenant

### DISCUSSION THREAD

**Discussion by hamna-python** [OrchardCMS/OrchardCore#14730] [Q&A]
I am experiencing an issue with a Blazor WebAssembly (WASM) module in a multi-tenant application setup. The application consists of two tenants: the default tenant and a custom-created tenant. The Blazor WASM module, which includes a counter component, has been integrated and enabled in both tenants. However, the module's behavior differs between the two tenants.

**Issue Details**
Default Tenant: In the default tenant, the Blazor counter component works correctly. It functions as intended on the index page and can be successfully added to and operated on other pages within this tenant.

Custom-Created Tenant: In the tenant that I created, the Blazor counter component only functions on the index page. When attempting to add and use this module on other pages within this tenant, it does not work.

**Expected Behavior**
The Blazor WASM module, including the counter component, should work consistently across all pages in both tenants, just as it does in the default tenant.

Default Tenant: 

index page:
![image](https://github.com/OrchardCMS/OrchardCore/assets/147750163/f2091552-c602-4fac-a15d-77c0f5fc8243)
about page:
![image](https://github.com/OrchardCMS/OrchardCore/assets/147750163/723dcb04-165e-41b5-b350-1b78c0286ee1)

Custom Tenant:
index page
![image](https://github.com/OrchardCMS/OrchardCore/assets/147750163/a0e7bd49-dc2f-4399-a5ae-155d916f5e96)

about page( here it is not working)

![image](https://github.com/OrchardCMS/OrchardCore/assets/147750163/da3dffce-094e-4a3a-abc9-0bc861321618)

**Comment by ns8482e** [comment]
@hamna-python most likely you have to update `<base>` url based on the Tenant prefix url

**Comment by ns8482e** [comment]
@hamna-python little old sample but should wok on .NET 6 https://github.com/surevelox/OrchardCore.Samples/tree/ns8482e/blazor-wasm/MultiTenantBlazorWasm
